src/Microsoft.Health.Fhir.Shared.Core/Features/Search/BundleFactory.cs

```diff
@@ -70,7 +70,9 @@ public ResourceElement CreateHistoryBundle(SearchResult result)
                     httpVerb = Bundle.HTTPVerb.PUT;
                 }
 #endif
-                resource.FullUrlElement = new FhirUri(_urlResolver.ResolveResourceWrapperUrl(r.Resource, true));
+
+                // resource.FullUrlElement = new FhirUri(_urlResolver.ResolveResourceWrapperUrl(r.Resource, true));
```

Comment:
nit: Should we remove this commented out line since it has been replaced?

Reply:
Done.